ui/src/js/profile/ToolsPage.js

```diff
@@ -0,0 +1,48 @@
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import Boostrap from 'bootstrap';
+import axios from 'axios';
+
+import Tools from './tools/Tools';
+
+export default class ToolsPage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+      },
+      pageCount: 0
+    };
+  }
+
+  callNextPage() {
+    var newPage = this.state.pageCount + 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
+  callPreviousPage() {
+    var newPage = this.state.pageCount - 1;
+    this.callPage(newPage);
+    this.setState({pageCount: newPage});
+  }
+
```

Comment:
i think these methods(callNext- and callPrevoiusPage) are related to the planted tree requests on the profile page,
so they can be removed here.

Reply:
Yes, I remember, we talked about it before my holidays. I removed the methods.